### DISCUSSION TITLE
I just got an checksum error while creating a PAR2 recovery file with MultiPar software 1.3.2

### DISCUSSION THREAD

**Discussion by icfsoft** [Yutaka-Sawada/MultiPar#120] [Q&A] [answered]
Windows memory diagnostic reports no error, as I am suspecting RAM issues.

What could this be? 

It has happened only once. I should have taken a screenshot.

@Yutaka-Sawada I found the log file.

**Comment by icfsoft** [comment]
18-Feb-24 2:04:28 PM
Parchive 2.0 client version 1.3.2.8 by Yutaka Sawada

Base Directory	: "..."
Recovery File	: "....par2"
CPU thread	: 6 / 16
CPU cache limit : 64 KB, 1024 KB
CPU extra	: x64 SSSE3 CLMUL AVX2
Memory usage	: Auto (20119 MB available)

Input File count	: 12
Input File total size	: 66692056758
Input File Slice size	: 22303244
Input File Slice count	: 2994
Recovery Slice count	: 346
Redundancy rate		: 11.55%
Recovery File count	: 1
Slice distribution	: 0, uniform (until 346)
Packet Repetition limit	: 0

  0.0% : Creating recovery slice
  0.1%
  0.2%
  0.3%
  0.4%
  0.5%
  0.6%
  0.7%
  0.8%
  0.9%
  1.0%
  1.1%
  1.2%
  1.3%
  1.4%
  1.5%
  1.6%
  1.7%
  1.8%
  1.9%
  2.0%
  2.1%
  2.2%
  2.3%
  2.4%
  2.5%
  2.6%
  2.7%
  2.8%
  2.9%
  3.0%
  3.1%
  3.2%
  3.3%
  3.4%
  3.5%
  3.6%
  3.7%
  3.8%
  3.9%
  4.0%
  4.1%
  4.2%
  4.3%
  4.4%
  4.5%
  4.8%
  5.2%
  5.5%
  5.8%
  6.2%
  6.5%
  6.8%
  7.1%
  7.5%
  7.8%
  8.1%
  8.5%
  8.8%
  9.1%
  9.5%
  9.8%
 10.1%
 10.5%
 10.8%
 11.1%
 11.5%
 11.8%
 12.1%
 12.4%
 12.8%
 13.1%
 13.4%
 13.8%
 13.9%
 14.0%
 14.1%
 14.2%
 14.3%
 14.4%
 14.5%
 14.6%
 14.7%
 14.8%
 14.9%
 15.0%
 15.1%
 15.2%
 15.3%
 15.4%
 15.5%
 15.6%
 15.7%
 15.8%
 15.9%
 16.0%
 16.1%
 16.2%
 16.3%
 16.4%
 16.5%
 16.6%
 16.7%
 16.8%
 16.9%
 17.0%
 17.1%
 17.2%
 17.3%
 17.4%
 17.5%
 17.6%
 17.7%
 17.8%
 17.9%
 18.0%
 18.1%
 18.2%
 18.3%
 18.4%
 18.8%
 19.1%
 19.5%
 19.8%
 20.1%
 20.5%
 20.8%
 21.1%
 21.5%
 21.8%
 22.1%
 22.5%
 22.8%
 23.1%
 23.5%
 23.8%
 24.2%
 24.5%
 24.8%
 25.1%
 25.5%
 25.8%
 26.2%
 26.5%
 26.8%
 27.1%
 27.5%
 27.7%
 27.8%
 27.9%
 28.0%
 28.1%
 28.2%
 28.3%
 28.4%
 28.5%
 28.6%
 28.7%
 28.8%
 28.9%
 29.0%
 29.1%
 29.2%
 29.3%
 29.4%
 29.5%
 29.6%
 29.7%
 29.8%
 29.9%
 30.0%
 30.1%
 30.2%
 30.3%
 30.4%
 30.5%
 30.6%
 30.7%
 30.8%
 30.9%
 31.0%
 31.1%
 31.2%
 31.3%
 31.4%
 31.5%
 31.6%
 31.7%
 31.8%
 31.9%
 32.0%
 32.1%
 32.2%
 32.3%
 32.6%
 33.0%
 33.3%
 33.6%
 34.0%
 34.3%
 34.6%
 34.9%
 35.3%
 35.6%
 35.9%
 36.3%
 36.6%
 36.9%
 37.2%
 37.6%
 37.9%
 38.2%
 38.5%
 38.9%
 39.2%
 39.5%
 39.9%
 40.2%
 40.5%
 40.9%
 41.2%
 41.5%
 41.6%
 41.7%
 41.8%
 41.9%
 42.0%
 42.1%
 42.2%
 42.3%
 42.4%
 42.5%
 42.6%
 42.7%
 42.8%
 42.9%
 43.0%
 43.1%
 43.2%
 43.3%
 43.4%
 43.5%
 43.6%
 43.7%
 43.8%
 43.9%
 44.0%
 44.1%
 44.2%
 44.3%
 44.4%
 44.5%
 44.6%
 44.7%
 44.8%
 44.9%
 45.0%
 45.1%
 45.2%
 45.3%
 45.4%
 45.5%
 45.6%
 45.7%
 45.8%
 45.9%
 46.0%
 46.1%
 46.5%
 46.8%
 47.1%
 47.5%
 47.8%
 48.1%
 48.4%
 48.8%
 49.1%
 49.5%
 49.8%
 50.1%
 50.4%
 50.7%
 51.1%
 51.4%
 51.7%
 52.0%
 52.4%
 52.7%
 52.9%
 53.2%
 53.6%
 53.8%
 54.1%
 54.5%
 54.8%
 55.1%
 55.5%
 55.6%
 55.7%
 55.8%
 55.9%
 56.0%
 56.1%
 56.2%
 56.3%
 56.4%
 56.5%
 56.6%
 56.7%
 56.8%
 56.9%
 57.0%
 57.1%
 57.2%
 57.3%
 57.4%
 57.5%
 57.6%
 57.7%
 57.8%
 57.9%
 58.0%
 58.1%
 58.2%
 58.3%
 58.4%
 58.5%
 58.6%
 58.7%
 58.8%
 58.9%
 59.0%
 59.1%
 59.2%
 59.3%
 59.4%
 59.5%
 59.6%
 59.7%
 59.8%
 59.9%
 60.0%
 60.3%
 60.6%
 60.9%
 61.2%
 61.6%
 61.9%
 62.2%
 62.5%
 62.8%
 63.2%
 63.5%
 63.8%
 64.1%
 64.4%
 64.7%
 65.1%
 65.4%
 65.7%
 66.1%
 66.4%
 66.7%
 67.1%
 67.4%
 67.7%
 68.0%
 68.3%
 68.6%
 69.0%
 69.3%
 69.4%
 69.5%
 69.6%
 69.7%
 69.8%
 69.9%
 70.0%
 70.1%
 70.2%
 70.3%
 70.4%
 70.5%
 70.6%
 70.7%
 70.8%
 70.9%
 71.0%
 71.1%
 71.2%
 71.3%
 71.4%
 71.5%
 71.6%
 71.7%
 71.8%
 71.9%
 72.0%
 72.1%
 72.2%
 72.3%
 72.4%
 72.5%
 72.6%
 72.7%
 72.8%
 72.9%
 73.0%
 73.1%
 73.2%
 73.3%
 73.4%
 73.5%
 73.6%
 73.7%
 73.8%
 73.9%
 74.2%
 74.5%
 74.9%
 75.2%
 75.5%
 75.8%
 76.1%
 76.4%
 76.8%
 77.1%
 77.4%
 77.7%
 78.1%
 78.4%
 78.7%
 79.1%
 79.4%
 79.7%
 80.0%
 80.3%
 80.6%
 81.0%
 81.3%
 81.6%
 81.9%
 82.2%
 82.6%
 82.9%
 83.2%
 83.3%
 83.4%
 83.5%
 83.6%
 83.7%
 83.8%
 83.9%
 84.0%
 84.1%
 84.2%
 84.3%
 84.4%
 84.5%
 84.6%
 84.7%
 84.8%
 84.9%
 85.0%
 85.1%
 85.2%
 85.3%
 85.4%
 85.5%
 85.6%
 85.7%
 85.8%
 85.9%
 86.0%
 86.1%
 86.2%
 86.3%
 86.4%
 86.5%
 86.6%
 86.7%
 86.8%
 86.9%
 87.0%
 87.1%
 87.2%
 87.3%
 87.4%
 87.5%
 87.6%
 87.7%
 87.8%
 87.9%
 88.0%
 88.1%
 88.2%
 88.3%
 88.7%
 89.0%
 89.4%
 89.7%
 90.1%
 90.5%
 90.8%
 91.2%
 91.6%
 91.9%
 92.3%
 92.6%
 93.0%
 93.3%
 93.7%
 94.0%
 94.4%
 94.7%
 95.1%
 95.4%
 95.8%
 96.1%
 96.5%
 96.8%
 94.3%
 94.5%
 94.7%
 94.9%
 95.0%
 95.1%
 95.2%
 95.4%
 95.5%
 95.6%
 95.7%
 95.8%
 95.9%
 96.0%
 96.1%
 96.2%
 96.3%
 96.4%
 96.6%
 96.7%
 96.8%
 96.9%
 97.0%
 97.1%
 97.2%
 97.3%
checksum mismatch, recovery slice 189

**Comment by Yutaka-Sawada** [comment, marked as answer]
Because it would be a hardware problem, I cannot help mostly. Some users reported this error on old PCs. Windows OS's memory diagnostic may not detect [Row hammer](https://en.wikipedia.org/wiki/Row_hammer) problem. You may need MemTest86 to check Row hammer. But, Row hammer should not happen on recent PCs normally. I could not solve this problem in software side.

You may change PC setting to be stable, such like over-clock or memory access speed. You may use the latest version MultiPar. If you enable GPU acceleration, just disable it. At MultiPar options, decrease CPU usage or disable CPU feature like AVX2. Though it becomes slow, slower calculation may avoid hardware error.

**Reply by icfsoft** [reply]
Thank you.

**Reply by icfsoft** [reply]
I just run MemTest86 V10.7 - 1 complete pass all tests - no errors
